runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -710,17 +717,17 @@ public void onWidgetLoaded(WidgetLoaded widgetLoaded)
 				int raidLevel = client.getVarbitValue(Varbits.TOA_RAID_LEVEL);
 				int teamSize =
 					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_0_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_1_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_2_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_3_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_4_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_5_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_6_HEALTH), 1) +
-					Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_7_HEALTH), 1);
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_1_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_2_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_3_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_4_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_5_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_6_HEALTH), 1) +
+						Math.min(client.getVarbitValue(Varbits.TOA_MEMBER_7_HEALTH), 1);
 				int raidDamage = client.getVarbitValue(Varbits.TOA_RAID_DAMAGE);
 				event = TOMBS_OF_AMASCUT;
 				container = client.getItemContainer(InventoryID.TOA_REWARD_CHEST);
-				metadata = new int[]{ raidLevel, teamSize, raidDamage };
+				metadata = new int[]{raidLevel, teamSize, raidDamage};
```

Comment:
Revert this whitespace change as well

Reply:
Reverted in b8175f742a5d8c62bd439918800fcf0b8a97015b

Reply:
You don't need to comment when you make these changes. We get notifications and can see the commit diffs.